### ISSUE TITLE
Unable to run bevy on Android SDK v 32

### ISSUE THREAD

**Issue by jbcurtin** [bevyengine/bevy#3458]
## Bevy version

0.5.0 from crate.io

## Operating system & version

macOS

## What you did

* Install Android SDK and NDK
* Follow bevy documentation to run on Android ( https://github.com/bevyengine/bevy/blob/main/examples/README.md#build--run )

## What you expected to happen

Android Debugging Session might have started

## What actually happened

Error Output:
```
MacBook-Pro bevy-android % RUST_BACKTRACE=full cargo apk run --example android
thread 'main' panicked at 'Failed to read source.properties: Os { code: 2, kind: NotFound, message: "No such file or directory" }', ~/.cargo/registry/src/github.com-1ecc6299db9ec823/ndk-build-0.4.3/src/ndk.rs:59:14
```

## Additional information

I reconize `cargo apk run --example android` is looking for `source.properties`. Which directory should I create the file or where is the URL which explains `source.properties`?

**Comment by mockersf** [comment]
I'm able to get something started on android, but I get a black screen and nothing happens. It's not complaining of a missing file for me

**Comment by tomadamatkinson** [comment]
I get a mixture of issues here.

Using Bevy Vanilla on the latest branch, the android app cant find the native lib and so crashes.

I updated bevy's winit dependencies to v0.26.0, syn to v1.0.86 and ndk-glue to v0.5.0; It builds correctly and the library can be found at runtime. But it is missing some symbols causing a crash at the link stage at androids onCreate.

Winit and ndk-glue using wgpu has worked for me recently and so I think these libraries aren't the issue - I moved to bevy to try speed things up. But I don't have enough experience with the Bevy Engine to give any more helpful information. It may be something at `bevy_main`.
